src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.liquid

```diff
@@ -35,11 +35,11 @@
 {% else -%}
     public {{ Class }}()
     {
-{% endif -%}
-{% if UseBaseUrl and GenerateBaseUrlProperty -%}
+{%     if UseBaseUrl and GenerateBaseUrlProperty -%}
         BaseUrl = "{{ BaseUrl }}";
+{%     endif -%}
```

Comment:
Something seems of with BaseUrl, the property isn't assigned due to this when run with NSwagStudio 13.20.0.0.

Reply:
Are you replacing NSwagStudio 13.20.0.0 templates with this one?

Can you share a simple repro?